src/grid/Cell.tsx

```diff
@@ -106,10 +106,11 @@ export default class Cell extends ThemedMixin(FocusMixin(WidgetBase))<CellProper
 					? w(TextInput, {
 							key: 'input',
 							theme,
-							classes,
+							classes: classes || {
```

Comment:
You're going to need to compose these together in all cases. Ideally we'd use `theme.compose`, but not available for class based widgets.

I believe you just need to merge at the `@dojo/widgets/text-input` level, something like:

```ts
const { classes = {} } = properties();
const passedInputClasses = classes['@dojo/widgets/text-input'] || {};
const newInputClasses = { 
   ...classes, 
   '@dojo/widgets/text-input': {
      ...passedInputClasses,
      input: [ this.theme(css.input) ]
   }
}
```

Reply:
Good call! I wasn't sure how best to handle the situation and I really didn't want to convert the whole grid in this ticket 😂